static/admin/config.yml

```diff
@@ -119,23 +119,23 @@ collections:
     editor:
       preview: false
     fields:
-      - { label: "Title", name: "resume-page-title", widget: "string" }
+      - { label: "Title", name: "resume-page-title", widget: "string",pattern: ['.{15}', "Must have at least 15 characters"] }
```

Comment:
Please add a max length to the page title at 50 or 60 characters (preferably 60 like on the blog page)
source:
https://moz.com/learn/seo/title-tag#:~:text=Optimal%20title%20length,your%20titles%20to%20display%20properly.